app/controllers/bookmarks_controller.rb

```diff
@@ -9,7 +9,9 @@ def index
 
   def new
     @bookmark            = Bookmark.new
-    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
+    query                = params[:query].to_s.squish
```

Comment:
It seems github was squishing for me as well :), I meant `name = "Some
very bad name"` (let's see if comments over mail work better:)

On Thu, Apr 7, 2016 at 2:03 PM, Ohad Levy <notifications@github.com> wrote:

> In app/controllers/bookmarks_controller.rb
> <https://github.com/theforeman/foreman/pull/3401#discussion_r58859517>:
>
> > @@ -9,7 +9,9 @@ def index
> >
> >    def new
> >      @bookmark            = Bookmark.new
> > -    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
> > +    query                = params[:query].to_s.squish
>
> On Thu, Apr 7, 2016 at 3:02 PM, Ivan Necas <notifications@github.com>
> wrote: name = "Some very bad name"
> not sure i follow? what exactly is the case? regardless, pushed to another
> issue is fine.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3401/files/4b2b563608bb239d97ab7de7ff0dc876eea79bd0#r58859517>
>